src/Components/FormContainer/FormContainer.tsx

```diff
@@ -19,6 +20,7 @@ export const FormContainer = () => {
   }
   return (
     <>
+    <ProgressBar questionsLength = {questions.length} answersLength ={answers.length}/>
```

Comment:
Using Question and answers length to keep track of user form progress.

Reply:
I like this! It will still measure the correct progress if a user skips a question, awesome job.